app/ui-react/packages/ui/src/Data/Virtualizations/VirtualizationCreateForm.tsx

```diff
@@ -0,0 +1,88 @@
+import { ActionGroup, 
+         Alert, 
+         Button, 
+         Card, 
+         CardBody, 
+         Form } from '@patternfly/react-core';
+import * as React from 'react';
+
+export interface IVirtualizationCreateValidationResult {
+  message: string;
+  type: 'danger' | 'success';
+}
+
+export interface IVirtualizationCreateFormProps {
+  /**
+   * The localized text for the cancel button.
+   */
+  i18nCancelLabel: string;
+
+  /**
+   * The localized text for the create button.
+   */
+  i18nCreateLabel: string;
+
+  /**
+   * `true` if work in progress and this form should disable user input.
+   */
+  isWorking: boolean;
+
+  /**
+   * Form level validationResults
+   */
+  validationResults: IVirtualizationCreateValidationResult[];
+
+  /**
+   * The callback fired when submitting the form.
+   * @param e
+   */
+  handleSubmit: (e?: any) => void;
+
+  /**
+   * The callback for cancel.
+   */
+  onCancel: () => void;
+}
+
+export class VirtualizationCreateForm extends React.Component<
```

Comment:
Can you make this a `FunctionComponent`?

Reply:
Good catch - I changed and pushed an update